checklist-model.js

```diff
@@ -106,6 +106,10 @@ angular.module('checklist-model', [])
 
       // exclude recursion
       tElement.removeAttr('checklist-model');
```

Comment:
I think the more elegant solution would be to change line 108 to the following:

tElement.removeAttr(tAttrs.$attr['checklistModel']);

This takes advantage of Angular's attribute normalization and covers all cases.